Makefile

```diff
@@ -0,0 +1,39 @@
+APPS = asset-manager content-store govuk-content-schemas multipage-frontend \
+	publishing-api router router-api rummager specialist-frontend \
+	specialist-publisher static travel-advice-publisher
+
+all: test
```

Comment:
@kevindew Actually, I think you're right. We're using `make` more as a task runner, usually you would have dependencies on files, but we can't do that in our case as containers aren't files.